amenity-points.mss

```diff
@@ -765,21 +769,34 @@
     marker-clip: false;
   }
 
-  [feature = 'historic_memorial'][zoom >= 17] {
+  [feature = 'historic_memorial'][zoom >= 17],
```

Comment:
You need to exclude here all defined memorials which are to be shown on z18 or z19, because now they first look like a generic memorial.

Reply:
What would be displayed at z17 ? If I exclude all the features listed below, we will highlight from z17 `historic=memorial` without memorial subtags which are not the most important ones.

Reply:
I made corrections, I have tested on `memorial=stele` and `memorial=bust` and will check the rest tomorrow